packages/cadl-ranch-specs/http/spread/main.tsp

```diff
@@ -0,0 +1,158 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@doc("Test for the spread operator.")
+@supportedBy("dpg")
+@scenarioService("/spread")
+namespace Spread;
+
+@doc("This is a simple model.")
+model ModelBodyParameter {
+  name: string;
+}
+
+@scenario
+@scenarioDoc("""
+Test case for spread named model. Should generate request body model named ModelBodyParameter.
+
+Expected request body:
+```json
+{ "name": "foo" }
+```
+""")
+@route("/model/request-body")
+@put
+op spreadModelAsRequestBody(...ModelBodyParameter): OkResponse;
```

Comment:
This is a valid scenario.

Must generate an operation and a model:
```
void spreadModelAsRequestBody(model_body_parameter: ModelBodyParameter)
```

Note that we don't have a parameter name in this scenario, I suggest it's guessed from the model name. See also:
https://github.com/Azure/typespec-azure/issues/2360